suite-native/module-device-onboarding/src/screens/DeviceAuthenticitySuccessScreen.tsx

```diff
@@ -0,0 +1,45 @@
+import { useCallback } from 'react';
+
+import { Button, PictogramTitleHeader, VStack } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import {
+    DeviceOnboardingStackParamList,
+    DeviceOnboardingStackRoutes,
+    StackProps,
+} from '@suite-native/navigation';
+
+import { DeviceOnboardingScreenWithExitButton } from '../components/DeviceOnboardingScreenWithExitButton';
+
+export const DeviceAuthenticitySuccessScreen = ({
+    navigation,
+}: StackProps<
+    DeviceOnboardingStackParamList,
+    DeviceOnboardingStackRoutes.DeviceAuthenticitySuccess
+>) => {
+    const handleContinueButtonPress = useCallback(() => {
+        navigation.navigate(DeviceOnboardingStackRoutes.DeviceTutorial);
+    }, [navigation]);
+
+    return (
+        <DeviceOnboardingScreenWithExitButton>
+            <VStack flex={1} spacing="sp32">
+                <VStack flex={1} spacing="sp40" justifyContent="center">
```

Comment:
Would a `Box` without `spacing` be sufficient here? What's the spacing for if there's only one element?